src/platform/src/test/java/org/geoserver/platform/resource/FileWrapperResourceTheoryTest.java

```diff
@@ -26,7 +30,13 @@
     protected Resource getResource(String path) throws Exception {
         File file = Paths.toFile(null, path);
         if (!file.isAbsolute()) {
-            file = Paths.toFile(folder.getRoot(), path);
+            if (folder.getRoot().getPath().startsWith("/") &&
+                    path.startsWith(folder.getRoot().getPath().substring(1))) {
+                //linux hack
```

Comment:
@NielsCharlier that makes sense. For the benefit of future maintainers, please change the comment to this (or similar):
```
// linux cannot detect absolute resource paths that do not start with a slash
```